packages/ui-tailwind/src/components/Block.tsx

```diff
@@ -0,0 +1,11 @@
+export type BlockProps = {
+  children?: React.ReactNode;
+  margin?: number;
+  padding?: number;
+};
+
+export const Block = ({ children, margin = 0, padding = 0 }: BlockProps) => {
+  const styles = `m-${margin} p-${padding} `;
```

Comment:
For this bit, style concatenation gets very annoying very quickly, there is phenomenal utility called classnames https://www.npmjs.com/package/classnames that allows you to make this much more scalable, I've used it in the ui package for the button, give it a go